code/world.dm

```diff
@@ -283,21 +283,13 @@ var/inerror = 0
 	else if (n > 0)
 		features += "~[n] player"
 
-	/*
-	is there a reason for this? the byond site shows 'hosted by X' when there is a proper host already.
-	if (host)
-		features += "hosted by <b>[host]</b>"
-	*/
-
 	if (!host && config && config.hostedby)
 		features += "hosted by <b>[config.hostedby]</b>"
 
 	if (features)
 		s += ": [jointext(features, ", ")]"
 
-	/* does this help? I do not know */
-	if (src.status != s)
-		src.status = s
```

Comment:
just a simple check to see if status already matched the new generated text to avoid resetting it I guess.

I decided there was no point since the generation is probably the most intensive part of it